Code/main.py

```diff
@@ -1,10 +1,10 @@
 #!/usr/bin/env python3
 
-__author__ = "SE Team Project 11 - HW 0"
+__author__ = "SE Team Project 11 - HW 1"
 __version__ = "0.1.0"
 __license__ = "MIT"
 
-
+# BEGIN - Older test functions - keeping them for CI sanity
```

Comment:
Pls remove older functions. We have a release for this. And HW1 was just about creating a good repo for which we will have new HW2 tests now